### ISSUE TITLE
Stuck on building project

### ISSUE THREAD

**Issue by 2joocy** [aws/lumberyard#406]
So I've been trying to get lumberyard up and running for the last 4 hours, switching between different solutions, although I cannot seem to build or create a new project. I've tried to let it sit for up to 45 minutes, but given my pc specs, I don't expect it to take that long.

I've tried to install both VS2015 and VS2017. I've edited project files according to the newest video put on the lumberyard youtube channel, and I've restarted multiple times.

![Project Configuration](https://i.imgur.com/dZzb8Fe.png)

The only thing logs are showing for the project, is the following:

```
[Warning] qrc:/markup/framework/AmazonFontLoader.qml:12:5: QML FontLoader: Cannot load font: "qrc:/fonts/OpenSans-Bold.ttf"(qrc:/markup/framework/AmazonFontLoader.qml, (null):12)
[Warning] Failed to load: qrc:/fonts/OpenSans-Bold.ttf(qrc:/markup/framework/AmazonFontLoader.qml, onStatusChanged:18)
[Info] Falling back to: function() { [code] }(qrc:/markup/framework/AmazonFontLoader.qml, onStatusChanged:22)
[Warning] qrc:/markup/framework/AmazonControls/AmazonDialog.qml:226:9: QML Item: Binding loop detected for property "height"(qrc:/markup/framework/AmazonControls/AmazonDialog.qml, (null):226)
[Warning] libpng warning: iCCP: known incorrect sRGB profile((no file), (null):0)
[Warning] libpng warning: iCCP: known incorrect sRGB profile((no file), (null):0)
[Warning] QWindow::setWindowState does not accept Qt::WindowActive((no file), (null):0)
[Critical] Failed to flush local socket.(d:\ly\workspace\PKG_v1_18\dev\Code\Tools\ProjectConfigurator\code\utilities\SingletonApplication.cpp, bool __cdecl ApplicationUtilities::SingletonApplication::BringPrimaryToFocus(void):164)
```

I have also installed MFC for VS, different windows builds, etc etc, running as admin, but nothing seems to help. If I enter some of the example levels, works just fine, I just can't create my own ones.

**Comment by bmk10** [comment]
do it in cmd. as long as solution .sln , in Solutions folder generated, lmf configure, lmbf build -all.. iin cmd.exe
cd to /lumberyard/1.19/dev and run https://docs.aws.amazon.com/lumberyard/latest/userguide/building-your-lumberyard-game-project.html

**Comment by AMZN-alexpete** [comment]
Hi @2joocy, please let me know if compiling from command line doesn't work for you.  Also, if you get an error when you run the configure command that your version of Visual Studio 2017 isn't detected try following these steps to update the  detected versions:

Open a command prompt and navigate to your Lumberyard dev folder
Run the command to generate a build settings file if you don't already have one

**lmbr_waf configure**

Open dev/_WAF_/user_settings.options with your favorite text editor
Find the following line which specifies the valid versions of visual studio 2017

**;win_vs2017_vswhere_args = -version [15.7.27703.2035,15.9.28307.665]**

Change it to this which allows all of 'em. NOTE: make sure to remove the semi colon at the beginning of the line

**win_vs2017_vswhere_args = -version [15.7.27703.2035,16]**

Back in your command window run

**lmbr_waf configure**

After this, your version of Visual Studio should be properly detected.

**Comment by wcbx** [comment]
It's really not a good idea to have a GUI interface building something which doesn't display console information and progress.

I've been waiting on a new project to build for almost an hour and I'm just going to give up if it isn't built after an error. It's spawning cl.exe processes so I assume it's compiling. It just takes too long to build.

**Comment by rumandcola** [comment]
Hi, I am having this issue as well and cant seem to get the resolution above to work or the one noted on the JennaDoll thread mentioned in the "How to Create and Compile a New Project for Lumberyard" video.  

I am making sure to use the Bin64vc141 ProjectConfigurator, I have updated the user_settings like above as well as using the recommendations in the mentioned video and enabled the IncrediBuild and user uber files.  Removed the semicolon as well to make sure the lines are enabled. 

lmbr_waf configure works but I havent had lmbr_waf build_win_x64_vs2017_profile -p game succeed yet.  Its bombed out 3 times at various spots.

If I try to build a project from the "front end" it just spins when I get to Building Project and this is all I see in the log

[Warning] qrc:/markup/framework/AmazonFontLoader.qml:12:5: QML FontLoader: Cannot load font: "qrc:/fonts/OpenSans-Bold.ttf"(qrc:/markup/framework/AmazonFontLoader.qml, (null):12)
[Warning] Failed to load: qrc:/fonts/OpenSans-Bold.ttf(qrc:/markup/framework/AmazonFontLoader.qml, onStatusChanged:18)
[Info] Falling back to: function() { [code] }(qrc:/markup/framework/AmazonFontLoader.qml, onStatusChanged:22)
[Warning] qrc:/markup/framework/AmazonControls/AmazonDialog.qml:226:9: QML Item: Binding loop detected for property "height"(qrc:/markup/framework/AmazonControls/AmazonDialog.qml, (null):226)
[Warning] libpng warning: iCCP: known incorrect sRGB profile((no file), (null):0)

I am using the installer from Amazon.  Not sure if trying to get it from here would be any different.

Any help?  I really want to dive into playing with Lumberyard but this is incredibly disheartening and defeating.

**Comment by joshbeacham** [comment]
@rumandcola  Please create a new issue. You can reference this one in your issue if you think they are related. You mentioned "Its bombed out 3 times at various spots." When you create your new issue, please provide additional information from your build logs to help diagnose. Thanks!

**Comment by AMZN-alexpete** [comment]
Closing this due to inactivity from the original issue creator.
